src/library/scala/annotation/nowarn.scala

```diff
@@ -14,15 +14,21 @@ package scala.annotation
 
 /** An annotation for local warning suppression.
   *
-  * The optional `value` parameter allows selectively silencing messages, see `scalac -Wconf:help`
-  * for help. Examples:
+  * The optional `value` parameter allows selectively silencing messages. See `-Wconf:help` for help
+  * on message filter expression, or use `@nowarn("verbose")` / `@nowarn("v")` to display message
```

Comment:
```suggestion
  * writing a message filter expression, or use `@nowarn("verbose")` / `@nowarn("v")` to display message
```
somehow I had trouble parsing "message filter expression", maybe a missing verbum?

Reply:
needless to add, I've used `-Wconf:help` so many times.